data/StoreSelectionModel.js

```diff
@@ -84,6 +84,13 @@ export class StoreSelectionModel {
         this.ids = clearSelection ? ids : union(this.ids, ids);
     }
 
+    /** Select all filtered records. */
+    @action
+    selectAll() {
+        this.select(this.store.records);
```

Comment:
Should we check the mode here? Looks like the behavior will be that this would effectively select the first record, which is kind of confusing.  Am torn between whether this should just be a no-op (and a warning logged?) for a 'single' mode sel model, or if we should throw. Seems like a programming error if `selectAll` is called for a non-multiple-mode sel model but throwing might be a little harsh in this case..